mmdet3d/models/detectors/centerpoint_two_stage.py

```diff
@@ -0,0 +1,175 @@
+# Copyright (c) OpenMMLab. All rights reserved.
+import torch
+from mmcv.runner import force_fp32
+from torch.nn import functional as F
+
+from mmdet3d.ops import Voxelization
+from mmdet.models import DETECTORS
+from .. import builder
+from .two_stage import TwoStage3DDetector
+
+
+@DETECTORS.register_module()
+class CenterPointTwoStage(TwoStage3DDetector):
+    """Two-stage center point.
+
+    Args:
+        voxel_layer (dict): Config of voxelization layer.
+        voxel_encoder (dict): Config of voxelization encoder.
+        middle_encoder (dict): Config of middle encoder.
+        backbone (dict): Config of one-stage backbone.
+        neck (dict): Config of one-stage neck.
+        rpn_head (dict): Config of one-stage head.
+        roi_head (dict): Config of two-stage head.
+        train_cfg (dict, optional): Config of the training. Defaults to None.
```

Comment:
it should be
```
        train_cfg (dict, optional): Config of the training. Defaults to None.

            - rpn (dict): Training config for RPN head.
            - rcnn (dict): Training config for two-stage roi head.
 ```
you can use this to verify the docstr http://livesphinx.herokuapp.com/

Reply:
same for test_cfg